DevEdu.API/DevEdu.Business/Services/LessonService.cs

```diff
@@ -117,26 +106,32 @@ public void UpdateStudentFeedbackForLesson(int lessonId, int userId, StudentLess
             studentLessonDto.Lesson = new LessonDto { Id = lessonId };
             studentLessonDto.User = new UserDto { Id = userId };
             _lessonRepository.UpdateStudentFeedbackForLesson(studentLessonDto);
+            return GetStudenLessonByLessonAndUserId(lessonId, userId);
         }
 
-        public void UpdateStudentAbsenceReasonOnLesson(int lessonId, int userId, StudentLessonDto studentLessonDto)
+        public StudentLessonDto UpdateStudentAbsenceReasonOnLesson(int lessonId, int userId, StudentLessonDto studentLessonDto)
         {
             studentLessonDto.Lesson = new LessonDto { Id = lessonId };
             studentLessonDto.User = new UserDto { Id = userId };
             _lessonRepository.UpdateStudentAbsenceReasonOnLesson(studentLessonDto);
+            return GetStudenLessonByLessonAndUserId(lessonId, userId);
         }
 
-        public void UpdateStudentAttendanceOnLesson(int lessonId, int userId, StudentLessonDto studentLessonDto)
+        public StudentLessonDto UpdateStudentAttendanceOnLesson(int lessonId, int userId, StudentLessonDto studentLessonDto)
         {
             studentLessonDto.Lesson = new LessonDto { Id = lessonId };
             studentLessonDto.User = new UserDto { Id = userId };
             _lessonRepository.UpdateStudentAttendanceOnLesson(studentLessonDto);
+            return GetStudenLessonByLessonAndUserId(lessonId, userId);
         }
 
         public List<StudentLessonDto> SelectAllFeedbackByLessonId(int lessonId)=>
             _lessonRepository.SelectAllFeedbackByLessonId(lessonId);
 
-        public StudentLessonDto GetStudenLessonByLessonAndUserId(int lessonId, int userId) =>
-            _lessonRepository.SelectByLessonAndUserId(lessonId, userId);
+        public StudentLessonDto GetStudenLessonByLessonAndUserId(int lessonId, int userId)
```

Comment:
you don't need this method
just use a direct call to _lessonRepository.SelectByLessonAndUserId(lessonId, userId) in places where you are callin this method